test/translate.js

```diff
@@ -16,3 +16,19 @@ test('Translate LISP to JavaScript', () => {
   const expected = 22;
   assert.strictEqual(result, expected);
 });
+
+test('Translate LISP condition to JavaScript', () => {
+  // TODO: replace with comparison operators when they are implemented
```

Comment:
I could use `eq` here but it doesn't transform properly https://github.com/metarhia/metalisp/issues/14